.evergreen/config.yml

```diff
@@ -2399,6 +2400,470 @@ axes:
+    AUTH: auth
+    SSL: ssl
+    TEST_SUITES: default_async
+    PYTHON_BINARY: C:/python/Python39/python.exe
+    SKIP_CSOT_TESTS: "true"
+- name: test-win64-py3.13-noauth-ssl-sync
+  tasks:
+    - name: .replica_set
+  display_name: Test Win64 py3.13 NoAuth SSL Sync
+  run_on:
+    - windows-64-vsMulti-small
+  expansions:
+    AUTH: noauth
+    SSL: ssl
+    TEST_SUITES: default
+    PYTHON_BINARY: C:/python/Python313/python.exe
+    SKIP_CSOT_TESTS: "true"
+- name: test-win64-py3.13-noauth-ssl-async
+  tasks:
+    - name: .replica_set
+  display_name: Test Win64 py3.13 NoAuth SSL Async
+  run_on:
+    - windows-64-vsMulti-small
+  expansions:
+    AUTH: noauth
+    SSL: ssl
+    TEST_SUITES: default_async
+    PYTHON_BINARY: C:/python/Python313/python.exe
+    SKIP_CSOT_TESTS: "true"
+- name: test-win64-py3.9-noauth-nossl-sync
+  tasks:
+    - name: .sharded_cluster
+  display_name: Test Win64 py3.9 NoAuth NoSSL Sync
+  run_on:
+    - windows-64-vsMulti-small
+  expansions:
+    AUTH: noauth
+    SSL: nossl
+    TEST_SUITES: default
+    PYTHON_BINARY: C:/python/Python39/python.exe
+    SKIP_CSOT_TESTS: "true"
+- name: test-win64-py3.9-noauth-nossl-async
+  tasks:
+    - name: .sharded_cluster
+  display_name: Test Win64 py3.9 NoAuth NoSSL Async
+  run_on:
+    - windows-64-vsMulti-small
+  expansions:
+    AUTH: noauth
+    SSL: nossl
+    TEST_SUITES: default_async
+    PYTHON_BINARY: C:/python/Python39/python.exe
+    SKIP_CSOT_TESTS: "true"
+- name: test-win32-py3.9-auth-ssl-sync
+  tasks:
+    - name: .standalone
+  display_name: Test Win32 py3.9 Auth SSL Sync
+  run_on:
+    - windows-64-vsMulti-small
+  expansions:
+    AUTH: auth
+    SSL: ssl
+    TEST_SUITES: default
+    PYTHON_BINARY: C:/python/32/Python39/python.exe
+    SKIP_CSOT_TESTS: "true"
+
+# Server tests for Win32.
+- name: test-win32-py3.9-auth-ssl-async
+  tasks:
+    - name: .standalone
+  display_name: Test Win32 py3.9 Auth SSL Async
+  run_on:
+    - windows-64-vsMulti-small
+  expansions:
+    AUTH: auth
+    SSL: ssl
+    TEST_SUITES: default_async
+    PYTHON_BINARY: C:/python/32/Python39/python.exe
+    SKIP_CSOT_TESTS: "true"
+- name: test-win32-py3.13-noauth-ssl-sync
+  tasks:
+    - name: .replica_set
+  display_name: Test Win32 py3.13 NoAuth SSL Sync
+  run_on:
+    - windows-64-vsMulti-small
+  expansions:
+    AUTH: noauth
+    SSL: ssl
+    TEST_SUITES: default
+    PYTHON_BINARY: C:/python/32/Python313/python.exe
+    SKIP_CSOT_TESTS: "true"
+- name: test-win32-py3.13-noauth-ssl-async
+  tasks:
+    - name: .replica_set
+  display_name: Test Win32 py3.13 NoAuth SSL Async
+  run_on:
+    - windows-64-vsMulti-small
+  expansions:
+    AUTH: noauth
+    SSL: ssl
+    TEST_SUITES: default_async
+    PYTHON_BINARY: C:/python/32/Python313/python.exe
+    SKIP_CSOT_TESTS: "true"
+- name: test-win32-py3.9-noauth-nossl-sync
+  tasks:
+    - name: .sharded_cluster
+  display_name: Test Win32 py3.9 NoAuth NoSSL Sync
+  run_on:
+    - windows-64-vsMulti-small
```

Comment:
We run 32-bit python on a 64-bit machine?

Reply:
32-bit versions of Python.  No one runs 32 bit machines anymore lol.

Reply:
But some apps still require 32 bit

Reply:
Just wanted to make sure, makes sense lol